code/modules/mob/mob_vars.dm

```diff
@@ -216,3 +215,44 @@
 	var/runechat_msg_location
 	/// The datum receiving keyboard input. parent mob by default.
 	var/datum/input_focus = null
+	/// Is our mob currently suiciding? Used for suicide code along with many different revival checks
+	var/suiciding = 0
+	/// Used for some screen objects, such as
+	var/list/screens = list()
+	/// lazy list. contains /obj/screen/alert only,  On /mob so clientless mobs will throw alerts properly
+	var/list/alerts
+	/// Makes items bloody if you touch them
+	var/bloody_hands = 0
+	/// Basically a lazy list, copies the DNA of blood you step in
+	var/list/feet_blood_DNA
+	/// affects the blood color of your feet, color taken from the blood you step in
+	var/feet_blood_color
+	/// Affects blood some more
+	var/blood_state = BLOOD_STATE_NOT_BLOODY
+	/// Doesn't really do much, xenos can't track blood and this doesn't seem to have much effect on anything
```

Comment:
These comments aren't really all that helpful, at least explain what they do in the autodoc and provide a comment explaining that they're useless alongside them.